src/guides/appchain-register.md

```diff
@@ -1,5 +1,59 @@
 ## Appchain Register
 
+### Appchain check items
+
+This section aims to list the items that the appchain team needs to check before appchain registration to ensure a smooth launch.
+
++ The appchain is using the latest version of the barnacle/barnacle-evm template
++ The version of Substrate used in Cargo.toml of appchain should be consistent with the template.
++ The runtime code should be consistent with the definition of the integrated Pallets in the template construct_runtime!
```

Comment:
The below pallets defined in the `construct_runtime!` part in the Appchain runtime code (runtime/src/lib.rs) should be consistent with the template.